grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2814,4 +2814,18 @@ class StockMovementService {
         }
         return true
     }
+
+    void deleteComment(Comment comment, Requisition requisition) {
+        requisition.removeFromComments(comment)
+    }
+
+    void saveComment(Comment comment) {
```

Comment:
I'm wondering if it should not return the `Comment`, so to be `Comment saveComment(Comment comment)` - usually persist methods return an entity, not void.
I know that anyway your reference to the comment in the controller will handle the eventual "persistence changes', but just saying.